deploy/scripts/src/mobile/README.md

```diff
@@ -23,8 +23,8 @@ npm install
 
 
 ## Run Command
-K6 environment variales can be found in test-mobile-journey.ts file.
-Profile and scenario can be found in test.ts file.
+K6 environment variales can be found in config.ts file.
```

Comment:
whilst we're here, please may we fix the spelling of variables

Reply:
Done